discovery/marathon/marathon.go

```diff
@@ -222,7 +222,8 @@ type DockerContainer struct {
 
 // Container describes the runtime an app in running in.
 type Container struct {
-	Docker DockerContainer `json:"docker"`
+	Docker       DockerContainer `json:"docker"`
+	PortMappings []PortMappings  `json:"portMappings"` // present in Marathon 1.5.x
```

Comment:
Comments should start with a capital letter and end in a full stop. I'd remove this comment, as the below ones cover this.

Reply:
updated